lib/draper/collection_decorator.rb

```diff
@@ -88,5 +98,10 @@ def item_decorator
         ->(item, options) { item.decorate(options) }
       end
     end
+
+    # Configures the strategy used to proxy the query methods, which defaults to `:active_record`.
+    def strategy
+      @strategy ||= LoadStrategy.new(Draper.default_query_methods_strategy)
```

Comment:
Does the name `LoadStrategy` still make sense in this new context? Should it be something like `QueryMethodStrategy` or something more specific to query methods? I don't have a big opinion here, but it does seem odd.